javascript-data-model/main.js

```diff
@@ -0,0 +1,37 @@
+/* exported orderHistory */
+
+var orderHistory = [
+  {
+    orderNum: '114-3941689-8772232',
+    orderPlaced: '08/04/2020',
+    orderDelivered: '08/08/2020',
+    productName: ['JavaScript for impatient programmers'],
+    productCost: [31.55],
```

Comment:
Rather than storing product cost in a separate array, it would probably be better to keep the information together. How could you keep an array of all items, but have more than one piece of information in each entry, such as product name, cost, and image?